File: src/FormiginationUI/wwwroot/js/Ops/Profile/Employment.js
var EmploymentManager = {
    saveEmployment: function () {

        var EmploymentObj = EmploymentHelper.createEmploymentObj();
        var jsonParam = 'EmploymentObj=' + EmploymentObj;
        var serviceUrl = "../Account/SaveEmployment/";
        AjaxManager.SaveObject(serviceUrl, jsonParam, onSuccess);


        function onSuccess(jsonData) {
            if (jsonData == "Success") {

                AjaxManager.MsgBox('success', 'center', 'Success:', 'Employment Info Save/Update Successfully',
                    [{
                        addClass: 'btn btn-primary',
                        text: 'Ok',
                        onClick: function ($noty) {
                            $noty.close();

                        }
                    }]);
            }
        }


    }
};

var EmploymentHelper = {
    init: function () {
        $("#btnEmployment").click(function () {
            EmploymentManager.saveEmployment();
        });
    },
    createEmploymentObj: function () {
        var obj = new Object();
        obj.GeneralAccount = $("#txtGeneralAccount").val();
        obj.EmploymentId = $("#txtEmploymentId").val();
        obj.CompanyName = $("#txtCompanyName").val();
        obj.CompanyBusiness = $("#txtCompanyBusiness").val();
        obj.CompanyLocation = $("#txtCompanyLocation").val();
        obj.Position = $("#txtPosition").val();
        obj.Department = $("#txtDepartment").val();
        obj.Responsibilities = $("#txtResponsibilities").val();
        obj.EmploymentDateFrom = $("#txtEmploymentDateFrom").val();
        obj.EmploymentDateTo = $("#txtEmploymentDateTo").val();
        obj.IsWorking = $("#txtIsWorking").val();

        return obj;
    },
};

File: Ops.ViewModels/Ops/Employment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpsModels.Ops
{
    public class Employment
    {
        public GeneralAccount GeneralAccount { get; set; }
        public int EmploymentId { get; set; }
        public string CompanyName { get; set; }

        public CompanyBusiness CompanyBusiness { get; set; }

        public string CompanyLocation { get; set; }
        public string Position { get; set; }
        public string Department { get; set; }
        public string Responsibilities { get; set; }
        public DateTime EmploymentDateFrom { get; set; }
        public DateTime EmploymentDateTo { get; set; }
        public bool IsWorking { get; set; }

    }
}
